addons/stock/models/stock_orderpoint.py

```diff
@@ -551,3 +551,6 @@ def _procure_orderpoint_confirm(self, use_new_cursor=False, company_id=None, rai
 
     def _post_process_scheduler(self):
         return True
+
+    def _get_orderpoint_procurement_date(self):
```

Comment:
I'm not sure about that it seems a bit risky IMO. I would rather add a context key to bypass company po_lead in the _compute_lead_days. What do you think of it?

Reply:
No bad idea, keep it like currently